client/components/credit-card/stored-card.tsx

```diff
@@ -1,25 +1,27 @@
 /**
  * External dependencies
  */
-import React from 'react';
+import React, { FunctionComponent } from 'react';
 import classNames from 'classnames';
-import PropTypes from 'prop-types';
-import { flowRight as compose } from 'lodash';
-import { localize } from 'i18n-calypso';
+import { useTranslate } from 'i18n-calypso';
 
 /**
  * Internal dependencies
  */
-import { withLocalizedMoment } from 'components/localized-moment';
+import { useLocalizedMoment } from 'components/localized-moment';
 
 /**
  * Style dependencies
  */
 import './stored-card.scss';
 
-export const getCreditCardSummary = ( translate, type, digits ) => {
+const getCreditCardSummary = (
+	translate: ReturnType< typeof useTranslate >,
+	type: string,
+	digits?: Props['lastDigits']
+) => {
 	const supportedTypes = {
-		[ 'american express' ]: translate( 'American Express' ),
+		'american express': translate( 'American Express' ),
```

Comment:
This needs some really thorough testing, wouldn't want to break anything 😄 Any idea who to ping for a domain-specific review here?

Reply:
👋 @dechov Can you help with testing/review or delegation thereof?